core/filter_form_api.php

```diff
@@ -2391,8 +2391,10 @@ function filter_form_draw_inputs( $p_filter, $p_for_screen = true, $p_static = f
 		if( $p_static) {
 			return $p_label;
 		} else {
-			if( $t_source_query_id > 0 ) {
-				$t_data_filter_id = ' data-filter_id="' . $t_source_query_id . '"';
+			if( filter_is_temporary( $t_filter ) ) {
```

Comment:
Should we use a different format for temp vs. permanent filters and use one filter id query string to represent both?  We can even have filter_get() handle both?

Reply:
in this PR, and due to inheriting part of that logic, the usage of a database saved filter is differentiated by using "filter_id" parameter, versus a temporary with "filter" parameter, on page requests.
This could be joined into a single value, for example, inferring that numerical ids are db-filters, and string ids are temporary filters.
This would fit in a more extensive work where the usage of filters across pages is cleaned up and improved (i commented on it in other reply).

A better way of dealing this has deeper implications that i'd rather leave out of this PR, but is due to be redesigned in the future.